app/components/time_ago_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Primer
+  # A basic wrapper for the "<time-ago>" element.
+  class TimeAgoComponent < Primer::Component
+    def initialize(time:, micro: false, **system_arguments)
+      @system_arguments = system_arguments
+      @system_arguments[:datetime] = time.utc.iso8601
+      @system_arguments[:classes] = class_names("no-wrap", @system_arguments[:classes])
```

Comment:
do we **always** want `no-wrap`?

Reply:
We had this in the helper/component on GitHub. I think we do. I can't think of a good reason to remove it.